docs/admin/users.md

```diff
@@ -1,77 +1,114 @@
+- `editor`
+- `reviewer`
+- `author`
+- `researcher`.
 
-On the Add User page, enter the following:
+These roles are separate from the [user roles assigned at a site-specific level](../sites/site_users.md), which allow registered users of the installation to have site access to build exhibit pages. 
+
+One example of how you may wish to manage the users of your site:
+
+- A **Global Admin** user who installs, manages, and upgrades the modules and themes, and controls fundamental installation settings and server utilities.
+- A **Supervisor** who manages user accounts on the installation and sites. 
+- One or more **Editors** who are responsible for the design and management of items, item sets, and resource templates. 
+- **Reviewer** staff members who are responsible for manually adding digitized or collected items and describing them according to institutional guidelines, and editing the additions of others. 
+- **Authors** who can add and edit their own items, and who are also given site-specific permissions to create exhibits and publish their research using items in the collection. 
+- **Researchers** who can be given site-specific permissions, but cannot add items. 
+
+## Create a user
 
-- *Name* (full name or credit name)
-- *Email address*
-- Select a *Role* using the dropdown menu (see above for user role privileges)
-- Check the *Is Active* box to confirm that the user is active.
+Only Supervisors and Global Administrators can create, edit, and delete users. Editors, Reviewers, Authors, and Researchers can edit their own user information but cannot change the information of other users. 
+
+To create a new user, select the "Add new user" button in the upper right hand corner of the Users section in the main administrative dashboard. 
 
 ![Add new user form with fields as described](adminfiles/users_addnew.png)
 
-## Managing Users
+On the Add User page, enter the following:
+
+- **Email address**
+- **Name** (full name or a display name)
+- A **Role** from the dropdown menu (see above for user role privileges)
+- Check the **Is active** box to confirm that the user is active. A user who is inactive cannot log in.
+
+The user is then sent a notification email to set up their account & password. The email will appear to come from what is set as the Administrator email in the [General settings of the installation](settings.md#general). Each email will expire 14 days after being sent. If the email expires, an administrator can manually set up a user's temporary password and email them. 
+
+> *Greetings!*
+>
+> *A user has been created for you on Omeka at [website]*
+>
+> *Your username is your email: [email]*
+>
+> *Click this link to set a password and begin using Omeka S: [website unique link]*
+>
+> *Your activation link will expire on [date and time]. If you have not completed the user activation process by the time the link expires, you will need to request another activation email from your site administrator.*
+
+!!! note
```

Comment:
Emails: the activations and forgot password emails are most common. Otherwise it's generally in the realm of plugins like you mention.

We could have a troubleshooting page/section just on email maybe.